packages/expo/src/__tests__/Haptic-test.ts

```diff
@@ -0,0 +1,92 @@
+import { UnavailabilityError } from 'expo-errors';
+import { NativeModules, Platform } from 'react-native';
+
+import {
+  mockPlatformAndroid,
+  mockPlatformWeb,
+  mockProperty,
+  unmockAllProperties,
+} from '../../test/mocking';
+import * as Haptic from '../Haptic/Haptic';
+
+describe('Haptic', () => {
+  describe('iOS', () => {
+    describe('selection()', () => {
+      it("Doesn't throw an error", async () => {
```

Comment:
- Read this as "it doesn't throw an error"
- The first word should be lowercase
- Prefer to use backticks for all the `it` descriptions since they are prose and we don't need to switch between single and double quotes then and it's pretty straightforward once you get used to it

```suggestion
      it(`doesn't throw an error`, async () => {
```